common/tier4_autoware_utils/test/src/geometry/test_geometry.cpp

```diff
@@ -735,6 +735,175 @@ TEST(geometry, transformPoint)
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.y, 0.4296803495383052);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.z, 0.40981009820187703);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.w, 0.76704600096616271);
+  }
+}
+
+TEST(geometry, inverseTransformPose)
+{
+  using tier4_autoware_utils::createQuaternionFromRPY;
+  using tier4_autoware_utils::deg2rad;
+  using tier4_autoware_utils::inverseTransformPose;
+
+  geometry_msgs::msg::Pose pose;
+  pose.position.x = 2.0;
+  pose.position.y = 4.0;
+  pose.position.z = 6.0;
+  pose.orientation = createQuaternionFromRPY(deg2rad(10), deg2rad(20), deg2rad(30));
+
+  // with transform
+  {
+    geometry_msgs::msg::Transform transform;
+    transform.translation.x = 1.0;
+    transform.translation.y = 2.0;
+    transform.translation.z = 3.0;
+    transform.rotation = createQuaternionFromRPY(deg2rad(30), deg2rad(30), deg2rad(30));
+
+    const geometry_msgs::msg::Pose pose_transformed = inverseTransformPose(pose, transform);
+
+    EXPECT_DOUBLE_EQ(pose_transformed.position.x, 0.11602540378443926);
+    EXPECT_DOUBLE_EQ(pose_transformed.position.y, 2.8325317547305482);
+    EXPECT_DOUBLE_EQ(pose_transformed.position.z, 2.4419872981077804);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.x, -0.17298739392508941);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.y, -0.08189960831908924);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.z, 0.029809019626209146);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.w, 0.98106026219040698);
+  }
+
+  // with pose_transform
+  {
+    geometry_msgs::msg::Pose pose_transform;
+    pose_transform.position.x = 1.0;
+    pose_transform.position.y = 2.0;
+    pose_transform.position.z = 3.0;
+    pose_transform.orientation = createQuaternionFromRPY(deg2rad(30), deg2rad(30), deg2rad(30));
+
+    const geometry_msgs::msg::Pose pose_transformed = inverseTransformPose(pose, pose_transform);
+
+    EXPECT_DOUBLE_EQ(pose_transformed.position.x, 0.11602540378443926);
+    EXPECT_DOUBLE_EQ(pose_transformed.position.y, 2.8325317547305482);
+    EXPECT_DOUBLE_EQ(pose_transformed.position.z, 2.4419872981077804);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.x, -0.17298739392508941);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.y, -0.08189960831908924);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.z, 0.029809019626209146);
+    EXPECT_DOUBLE_EQ(pose_transformed.orientation.w, 0.98106026219040698);
+  }
+}
+
+TEST(geometry, inverseTransformPoint)
+{
+  using tier4_autoware_utils::createQuaternionFromRPY;
+  using tier4_autoware_utils::deg2rad;
+  using tier4_autoware_utils::inverseTransformPoint;
+  using tier4_autoware_utils::inverseTransformPose;
+
+  geometry_msgs::msg::Pose pose_transform;
+  pose_transform.position.x = 1.0;
+  pose_transform.position.y = 2.0;
+  pose_transform.position.z = 3.0;
+  pose_transform.orientation = createQuaternionFromRPY(deg2rad(30), deg2rad(30), deg2rad(30));
+
+  // calc expected valuea
```

Comment:
typo?

Reply:
yes, fixed in https://github.com/autowarefoundation/autoware.universe/pull/1257/commits/8d0da87019b4d85aba4416b99c8d8c92e0c8856f